packages/plugins/rubygem-elastic-apm/rubygem-elastic-apm.spec

```diff
@@ -0,0 +1,98 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
```

Comment:
I guess this has been generated from the `scl` template, but the file is missing the `# template: scl` header, could you add it (back) please to make gem updates easier?